specification/compute/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/examples/ContainerGroupsListAll.json

```diff
@@ -8,50 +8,71 @@
       "body": {
         "value": [
           {
-            "id": "/subscriptions/subid/resourceGroups/demo/providers/Microsoft.ContainerInstance/containerGroups/demo1",
-            "location": "west us",
-            "name": "demo1",
-            "properties": {
```

Comment:
This is the output when I test in with CLI. I think it's because I added `"x-ms-client-flatten": true`